tests/cli.test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash 
+export SCAFFOLD_NO_CLOBBER="true" 
+export SCAFFOLD_OUT="gen" 
+export SCAFFOLD_DIR=".scaffold,.examples" 
+
+go run main.go new --test cli
```

Comment:
I think we'll need more than one test case. Let's say this example had a couple of "features". Then I would need to run `new` to scaffold out several entries in the matrix like `{featureA, featureB, featureA+B}` to have test coverage for mistakes in the scaffold that can only be observed when those features interact in the generated app, as well as verify each feature works in isolation.

So maybe the yaml provides named test cases? Then I would reference the names here like  `--test=case1`?